### PR TITLE
Add keynoters to schedule

### PR DESCRIPTION
**Pull request by williln**
Doesn't solve all style issues but gets their names on the schedule!

### CODE REVIEW

File: _schedule/talks/2017-08-14-10-00-keynote.md [hunk 1]
```diff
@@ -6,15 +6,12 @@ layout: session-details
 permalink: /talks/keynote-1/
 sitemap: false
 room: full
-title: Opening and Keynote
+title: Opening and Keynote, Alicia V. Carr
```
Comment (jefftriplett) [inline _schedule/talks/2017-08-14-10-00-keynote.md]: Do the keynotes have titles?

File: _schedule/talks/2017-08-15-10-00-keynote.md [hunk 2]
```diff
@@ -6,15 +6,12 @@ layout: session-details
 permalink: /talks/keynote-2/
 sitemap: false
 room: full
-title: Keynote
+title: Keynote, Ana Balica
```
Comment (jefftriplett) [inline _schedule/talks/2017-08-15-10-00-keynote.md]: Ditto

**Review by jefftriplett — CHANGES_REQUESTED** [review]
Please see the comments about the title vs. the Keynoter's name. Both will show up post schedule format changes.

**Comment by williln** [comment]
@jefftriplett Added keynote titles, added abstracts so the keynote detail pages are ready. One caveat: the keynotes on the schedule now render like this: 

screen shot 2017-07-18 at 10 46 07 am

The detail page, though, looks like the other talk pages! 

screen shot 2017-07-18 at 10 46 17 am

**Comment by williln** [comment]
@jefftriplett Also pushed a change to remove speaker's name from title. Sorry; it was hard for me to see what would/would not show up w/o the styling being consistent.

**Comment by jefftriplett** [comment]
I'll see if I can fix it real quick. Thanks for the update.

**Review by jefftriplett — APPROVED** [review]

**Pull request merged**